### PR TITLE
[ONNX] Export det

### PR DESCRIPTION
**Pull request by neginraoof**
Added symbolic to export det in opset 11
Updating ONNX submodule is required for det export

### CODE REVIEW

File: test/onnx/test_pytorch_onnx_onnxruntime.py [hunk 1]
```diff
@@ -1161,6 +1161,15 @@ def forward(self, x):
         x = torch.tensor([0.9920, -1.0362, -1.5000, 3.5000], requires_grad=True)
         self.run_test(Round(), x)
 
+    @skipIfUnsupportedMinOpsetVersion(11)
+    def test_det(self):
+        class Det(torch.nn.Module):
+            def forward(self, x):
+                return torch.det(x)
+
+        x = torch.randn(2, 3, 5, 5, requires_grad=True)
```
Comment (BowenBao) [inline test/onnx/test_pytorch_onnx_onnxruntime.py]: Can we add another test case where input shape is (5, 5)? That covers the case where output is scalar.

**Review by BowenBao — APPROVED** [review]
LGTM, minor comment on better test coverage.

**Comment by lara-hdr** [comment]
@neginraoof TestOperators.test_det is failing

**Review by houseroad — CHANGES_REQUESTED** [review]
Export looks good.

`RuntimeError: lu: LAPACK library not found in compilation`

Please fix the tests.

**Comment by neginraoof** [comment]
@houseroad I didn't see this locally. Trying to repro. Do you have an idea why it complains about the missing package?

**Comment by neginraoof** [comment]
cc @houseroad for review

**Review by houseroad — COMMENTED** [review]
Fix the conflict?

**Comment by neginraoof** [comment]
@houseroad just updated. please review

**Review by houseroad — CHANGES_REQUESTED** [review]
Please do NOT update onnx submodule

**Comment by neginraoof** [comment]
@houseroad onnx is reset. Just rebased this

**Comment by neginraoof** [comment]
@houseroad this is ready

**Comment by neginraoof** [comment]
@houseroad please review the updates

**Review by houseroad — APPROVED** [review]
LGTM

**Pull request closed without merging**